packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -283,22 +283,40 @@ export abstract class AbstractMessageManager<
   }
 
   /**
-   * Sets the message to a new status via a call to this.setMsgStatus and
-   * updates the rawSig field in this.messages.
+   * Sets the message via a call to this.setResult and updates status of the message.
    *
    * @param messageId - The id of the Message to sign.
    * @param rawSig - The data to update rawSig in the message.
    * @param status - The new message status.
    */
   setMessageStatusAndResult(messageId: string, rawSig: string, status: string) {
+    this.setResult(messageId, rawSig, false);
+    this.setMessageStatus(messageId, status);
+  }
+
+  /**
+   * Sets the message result.
+   *
+   * @param messageId - The id of the Message to sign.
+   * @param result - The data to update result in the message.
+   * @param updateMessage - Whether to update the message in the state.
+   */
+  setResult(
+    messageId: string,
+    result: string,
+    /* istanbul ignore next */
```

Comment:
Ignored this and the below branch because setResult is only in the use of falsy value in the class.